### PR TITLE
swirc: update to 3.3.7.

### PR DESCRIPTION
**Pull request by uhlin**
Swirc 3.3.7 out

### CODE REVIEW

File: srcpkgs/swirc/template [hunk 1]
```diff
@@ -15,17 +17,15 @@ license="BSD-3-Clause, ISC, MIT"
...
-	vsed -i "${wrksrc}/src/events/privmsg.cpp" -e "s,/usr/local/share/swirc,/usr/share/swirc,"
-	vsed -i "${wrksrc}/src/main.cpp" -e "s,/usr/local/share/locale,/usr/share/locale,"
-}
+build_options_default="notify"
 
 post_configure() {
-	local _file="options.mk"
+	local _file
+
+	_file="options.mk"
```
Comment (classabbyamp) [inline srcpkgs/swirc/template:28]: this change isn't necessary, it doesn't do anything different

**Comment by classabbyamp** [comment]
was this tested on void? you didn't fill out the PR template

**Comment by uhlin** [comment]
Yeah, I tested it on Void. I run Void on one of my servers. Where can I find the PR template?

File: srcpkgs/swirc/template (hunk 1 above)
Comment (uhlin) [reply srcpkgs/swirc/template:28]: True :)

**Comment by classabbyamp** [comment]
it should be what's in the comment box when making a new pr, but it's also available [here](https://github.com/void-linux/void-packages/raw/master/.github/pull_request_template.md)

**Comment by uhlin** [comment]
Aha... must've missed it (not on purpose). Next time I'll do it. But I tested it on my native architecture amd64. And built it successfully for the other architectures.

**Pull request merged**